packaging/PLATFORM_SUPPORT.md

```diff
@@ -0,0 +1,153 @@
+<!--
+title: "Netdata platform support policy"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/PLATFORM_SUPPORT.md
+-->
+
+# Netdata platform support policy
+
+Netdata defines three tiers of official support: Core, Intermediate, and Community. Each tier defines different
+guarantees for platforms in that tier, as follows:
+
+- **Core**: Platforms in the core support tier are our top priority. They are covered rigorously in our CI, usually
+  include official binary packages, and any platform specific bugs recieve a high priority. From the perspective
+  of our developers, platforms in the core support tier _must_ work, with almost no exceptions.
+- **Intermediate**: Platforms in the intermediate support tier are those which Netdata wants to support, but cannot
+  justify core level support for. They are also covered in CI, but not as rigorously as the core tier, may or may
+  not include official binary packages, and any platform specific bugs recieve a normal priority. This is generally
+  where we will add new platforms we have decided to officially support ourselves.
+- **Community**: Platforms in the community support tier are those which are primarily supported by community
+  contributors. They may recieve some support from Netdata, but are only a best-effort affair. When a community
+  member makes a contribution to add support for a new platform, that platform generally will start in this tier.
+
+Additionally, we define two categories for special cases that we do not support:
+
+- **Third-party Supported Platforms**: Some platform maintainers actively support Netdata on their platforms even
```

Comment:
It's not clear what is the difference between 3rd party supported platforms & community platforms.

Reply:
The key distinction is that the kickstart script and/or our static builds are not expected to work on platforms with purely third-party support, while they are expected to work on community supported platforms.

NixOS is a prime example. The kickstart script _does not work_ there at all (and we can’t really make it work properly there either), but the NixOS team packages Netdata and supports it on NixOS.

This is essentially a way for us to officially say ‘we know Netdata kind of works here, but any issues should be taken to the relevant third-party maintainers’. I would like this specifically codified in our support policy so we can redirect users with issues on these platforms without having to explain things in detail every time.